Dockerfile

```diff
@@ -1,108 +1,115 @@
+# syntax=docker/dockerfile:1
+
 # This Dockerfile builds the docs for https://docs.docker.com/
 # from the master branch of https://github.com/docker/docker.github.io
-#
-# Here is the sequence:
-# 1.  Set up base stages for building and deploying
-# 2.  Collect and build the reference documentation (from upstream resources)
-# 3.  Build static HTML from the current branch
-# 4.  Build the final image, combining the reference docs and current version
-#     of the documentation
-#
-# When the image is run, it starts Nginx and serves the docs at port 4000
-
-# Jekyll environment (development/production)
+
+# Use same ruby version as the one in .ruby-version
+# that is used by Netlify
+ARG RUBY_VERSION=2.6.10
+# Same as the one in Gemfile.lock
+ARG BUNDLER_VERSION=2.3.13
+
 ARG JEKYLL_ENV=development
+ARG DOMAIN=docs.docker.com
 
-# Engine
 ARG ENGINE_BRANCH="20.10"
```

Comment:
Should we use the new format to describe arguments (to allow the `docker buildx outline` feature to describe things?)

Reply:
Yes we could but these args will be removed in this follow-up if we are ok with it. See https://github.com/docker/docker.github.io/pull/14688/commits/c20f8d42db17428e36a01ab1f198aeff0047c759

Reply:
As discussed, will move the [JSON config for the plugin](https://github.com/docker/docker.github.io/blob/bd1eac0b7b0e413e4c260ea8f97c0e284a272080/_data/fetch-remote.json) in #14688 to the `_config.yml`